util-logging/src/test/scala/com/twitter/logging/FileHandlerTest.scala

```diff
@@ -285,7 +285,7 @@ class FileHandlerTest extends AnyWordSpec with TempFolder {
 
         val handler = FileHandler(
           filename = folderName + "/test.log",
-          rollPolicy = Policy.MaxSize(maxSize.bytes),
+          rollPolicy = Policy.MaxSize(maxSize.toLong.bytes),
```

Comment:
This conversion is necessary in Scala3 because the extension method is only defined on `Long`, and it no longer implicitly converts to `Int`. In #290 I added an implicit conversion as forwarder but that's not landed in `develop` yet.

Reply:
I can make a PR based on what I've already added in #290 to keep source compatibility for Scala3

Reply:
The separate PR is here: #296